tests/lib/rules/lines-around-comment.js

```diff
@@ -317,8 +317,195 @@ ruleTester.run("lines-around-comment", rule, {
                 allowBlockEnd: true
             }],
             ecmaFeatures: {classes: true}
-        }
+        },
+
+        // check for object start comments
```

Comment:
Simple ones are fine, for the complicated ones its always preferred to have multi-line.